client/beefy/src/lib.rs

```diff
@@ -106,18 +110,21 @@ where
 }
 
 /// BEEFY gadget initialization parameters.
-pub struct BeefyParams<B, BE, C, N>
+pub struct BeefyParams<B, BE, C, N, R>
 where
 	B: Block,
 	BE: Backend<B>,
 	C: Client<B, BE>,
-	C::Api: BeefyApi<B>,
+	R: ProvideRuntimeApi<B>,
+	R::Api: BeefyApi<B>,
```

Comment:
This is needed because test client doesn't impl `ProvideRuntimeApi`.

If I get these tests working I will look for a better way that avoids adding this extra generic (maybe just have test client impl some customizable impl `ProvideRuntimeApi` mock).